spark-connect/client/src/test/scala-spark-master/io/delta/connect/tables/DeltaTableBuilderSuite.scala

```diff
@@ -361,13 +360,16 @@ class DeltaTableBuilderSuite extends DeltaQueryTest with RemoteSparkSession {
   test("errors if table name and location are different paths") {
     withTempPath { dir =>
       val path = dir.getAbsolutePath
+      // TODO: This should be an AnalysisException, but it ends up as an Exception in Spark 
+      // arising from the connect client attempting to enrich the exception with a Delta error code. 
       val e = intercept[Exception] {
         io.delta.tables.DeltaTable.create(spark).tableName(s"delta.`$path`")
           .addColumn("c1", "int")
-          .location("src/test/resources/delta/dbr_8_0_non_generated_columns")
+          .location("src/test/resources/delta/non_generated_columns")
           .execute()
       }
-      assert(e.getMessage.contains("DELTA_CREATE_TABLE_IDENTIFIER_LOCATION_MISMATCH"))
+      val deltaErrorCode = "DELTA_CREATE_TABLE_IDENTIFIER_LOCATION_MISMATCH"
```

Comment:
```suggestion
      val deltaErrorClass = "DELTA_CREATE_TABLE_IDENTIFIER_LOCATION_MISMATCH"
```

To follow the naming of Exception arguments